### DISCUSSION TITLE
User active or not

### DISCUSSION THREAD

**Discussion by froelants** [timmo001/system-bridge#2711] [Ideas] [answered]
I would like to turn on my disk light (by Home assistant) when I'm active using my PC, therefor a state "active user" would be useful in the sensor system-bridge. Knowing who is logged in, is an option for my use case.
The state "active user" should go to false when the user logs out or the screensaver turn on, perhaps that could be an other state. 
My computer is most of the time running day and night so I cannot user the sensor that tells me the computer is on.

**Comment by willoma** [comment]
Hello,

I have a similar need, to track the time my kids spend on their computer.

**Comment by timmo001** [comment, marked as answer]
https://github.com/timmo001/system-bridge-backend/pull/18
